From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Fri, 12 Jun 2015 15:50:22 -0400
Subject: print config option on startup

---
 caPutLogApp/caPutLog.c | 17 +++++++++++++++--
 1 file changed, 15 insertions(+), 2 deletions(-)

diff --git a/caPutLogApp/caPutLog.c b/caPutLogApp/caPutLog.c
index 7573f7a..ebbaa6a 100644
--- a/caPutLogApp/caPutLog.c
+++ b/caPutLogApp/caPutLog.c
@@ -69,9 +69,22 @@ int epicsShareAPI caPutLogInit (const char *addr_str, int config)
 {
     int status;
 
-    if (config == caPutLogNone) {
+    switch(config) {
+    case caPutLogNone:
+        printf("caPutLogInit config: Disabled\n");
         return caPutLogSuccess;
-        errlogSevPrintf(errlogInfo, "caPutLog: disabled\n");
+    case caPutLogOnChange:
+        printf("caPutLogInit config: OnChange\n");
+        break;
+    case caPutLogAll:
+        printf("caPutLogInit config: All\n");
+        break;
+    case caPutLogAllNoFilter:
+        printf("caPutLogInit config: AllNoFilter\n");
+        break;
+    default:
+        printf("caPutLogInit config: Unknown (must be -1, 0, 1, or 2)\n");
+        return caPutLogError;
     }
 
     status = caPutLogClientInit(addr_str);
